packages/flutter/test/widgets/gesture_detector_test.dart

```diff
@@ -125,6 +125,32 @@ void main() {
     expect(didEndPan, isTrue);
   });
 
+  testWidgets('DragEndDetails returns the last known position', (WidgetTester tester) async {
+    Offset updateOffset = const Offset(10, 10);
+    Offset? endOffset;
+
+    await tester.pumpWidget(
+      GestureDetector(
+        onPanStart: (DragStartDetails details) {
+        },
+        onPanUpdate: (DragUpdateDetails details) {
+          updateOffset += details.delta;
+        },
+        onPanEnd: (DragEndDetails details) {
+          endOffset = details.localPosition;
```

Comment:
Nit: Is it worth it to also test details.globalPosition and show that it's not the same as localPosition? Or in this case it might be the same, so you might need a separate test.